src/components/MembersPanel.tsx

```diff
@@ -368,9 +368,31 @@ const MembersPanel = ({ members, setMembers }: MembersPanelProps) => {
       );
    }
 
+   function totalMembers(): number {
+      return (
+         members.length
+      );
+   }
+
+   function activeMembers(): number {
+      let activeTotal = 0
+
+      for (let member of members) {
+         if (member.active) {
+            activeTotal += 1
```

Comment:
```suggestion
            activeTotal ++;
```
let's use ++ notation here